### ISSUE TITLE
Accordion Component

### ISSUE THREAD

**Issue by buseselvi** [Trendyol/baklava#739]

**Comment by fatihhayri** [comment]
Native accordion

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details
https://open-ui.org/components/accordion.explainer
https://12daysofweb.dev/2021/details-summary/

**Comment by fatihhayri** [comment]
@buseselvi, is the title any hover action?

**Comment by buseselvi** [comment]
@fatihhayri yes, I added hover in the accordion list variant at the bottom of the document.

**Comment by fatihhayri** [comment]
I made accordion component with native HTML element `<details>` and `<summary>`.

https://codepen.io/fatihhayri/pen/bGzYYWv?editors=1100

Note: name attribute only Chrome 120, Safari 17.2. support. (this time)

**Comment by AykutSarac** [comment]
@Enes5519 do we have progress on this issue?

**Comment by AykutSarac** [comment]
Great work, Enes! Here are some suggestions for the ADR:

* The 'accordion' prefix is not needed for the event name **bl-accordion-toggle**.
* I believe we can omit the variant for **bl-accordion-group**. In instances of standalone usage, there doesn't seem to be a need to enclose accordions within the accordion group. We can simply use **bl-accordion-group** for lists and **bl-accordion** for standalone scenarios.
* The option to make an accordion either standalone or within a list may be marked as `disabled`, although this detail is not reflected in the Figma. We can explore this further during the design meeting with @buseselvi.
* We could consider renaming `auto-collapse` to a shorter alternative like `multiple`for simplicity.

**Comment by buseselvi** [comment]
I added the disabled versions to the design guide. Thank you @AykutSarac 🙏🏻

**Comment by Enes5519** [comment]
Thx for review @AykutSarac 

1. Done ✅ 
2. I added it in case you want auto-collapse in your accordions and also want them to stand apart. We can change the name of the standalone variant if it creates confusion.
3. Done ✅  (Thx @buseselvi )
4. `multiple` does not explain what it does, I can update it if there is an alternative.

**Comment by AykutSarac** [comment]
> Thx for review @AykutSarac
> 2. I added it in case you want auto-collapse in your accordions and also want them to stand apart. We can change the name of the standalone variant if it creates confusion.
> 4. `multiple` does not explain what it does, I can update it if there is an alternative.

2. I'm not sure if it makes sense to auto-collapse if they are standing apart, I mean they should be using the list in this case. So, I offered to use `bl-accordion-group` only for lists without the variant. We can discuss this further at the weekly update.
4. Agreed, couldn't think of anything better. `auto-collapse` sounds ok.

**Comment by Enes5519** [comment]
2. Separate accordions will need to write their own for auto-collapse. We talked with @buseselvi, but we can also bring it to the weekly agenda.
3. thx

**Comment by Enes5519** [comment]
I updated it as we discussed at the weekly meeting.

- Remove standalone variant from bl-accordion-group.

**Comment by Enes5519** [comment]
Since auto-collapse is enabled by default, it is not possible to turn it off as an attribute, so I will change the name to keep-open. @buseselvi @AykutSarac
